### ISSUE TITLE
Cant Download from files

### ISSUE THREAD

**Issue by Alexis6307** [ra1nb0w/mikrotik-winbox-snap#1]
Hello,

Winbox snap package can't access filesystem to download files.

**Comment by speakradmin** [comment]
Yep, also can't access the file system which makes importing and exporting .WBX files impossible. I also tried moving files to the .wine directory inside the snap folder and although the files then appear in the files dialog, they still don't get imported.

**Comment by ra1nb0w** [comment]
I did an update that should fix this, try with
```
sudo snap install --edge mikrotik-winbox 
```
In the meantime, I rebuild the snap with new wine package but to become stable I am waiting that the interfaces link automatically.

**Comment by ra1nb0w** [comment]
any news about upstream?

**Comment by LinuxCuba** [comment]
Please fix the problem, i need winbox in ubuntu os run in wine.  Thank.

sudo snap install --edge mikrotik-winbox
error: snap "mikrotik-winbox" not found

**Comment by six2dez** [comment]
> Please fix the problem, i need winbox in ubuntu os run in wine. Thank.
> 
> sudo snap install --edge mikrotik-winbox
> error: snap "mikrotik-winbox" not found

Don't use the snap package, Winbox always runs great under Wine.
